modules/apps/bookmarks/bookmarks-web/src/main/resources/META-INF/resources/bookmarks/js/BookmarksManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function propsTransformer({
+	additionalProps: {deleteEntriesURL, inputId, inputValue, trashEnabled},
+	portletNamespace,
+	...props
+}) {
+	const deleteEntries = () => {
+		if (
+			trashEnabled ||
+			confirm(
+				Liferay.Language.get(
+					'are-you-sure-you-want-to-delete-the-selected-entries'
+				)
+			)
+		) {
+			const form = document.getElementById(`${portletNamespace}fm`);
+
+			if (!form) {
+				return;
+			}
```

Comment:
Do we want to wrap the `if (form)` check like this or would you rather wrap the entire code block below in the `if (form)` check?

Ex.
```
if (form) {
	form.setAttribute('method', 'post');

	const inputElement = document.getElementById(
		`${portletNamespace}${inputId}`
	);

	if (inputElement) {
		inputElement.setAttribute('value', inputValue);

		submitForm(form, deleteEntriesURL);
	}
}
```

Reply:
@jonmak08 Here was [the glorious moment](https://github.com/liferay/liferay-frontend-guidelines/issues/6) when we overwhelmingly supported dropping the single return policy :slightly_smiling_face: 

For me, the longer the `if` block, the more it makes sense for an early return. But, it's basically up to developer, I'd be fine to accept either.

Reply:
Thanks for the update @markocikos!

Just to keep things consistent, I will side on the early return statements.  Trying to keep things as consistent within our JS code as possible to make it easier to work and understand instead of allowing multiple/differing patterns throughout our code base.